src/sections/Community/Handbook/faq.js

```diff
@@ -16,61 +15,191 @@ const Faq = () => {
+              help. Alternatively, Look to find the{" "}
+              <a href="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22help+wanted%22+">
+                help-wanted
+              </a>{" "}
+              issues.
+            </p>
+
+            <h4>
+              Q: In addition to the routinely scheduled project meetings, do
+              project contributors ever meet separately to deep-dive on special
+              topics?
+            </h4>
+            <p>
+              A: Yes, outside of the regularly scheduled meetings on the
+              community calendar (meet.layer5.io), there are any number of other
+              meetings that occur to facilitate deeper levels or more
+              concentrated attention to particular bodies of work (e.g. a design
+              spec review). On occasion, these one-off meetings are scheduled in
+              advance and put onto the community calendar, while other other
+              times they are simply self-organized by interested contributors.
+              Often contributors will use Slack huddles or Zoom meetings to
+              facilitate the discussion. If you need a new meeting placed onto
+              the community calendar, please reach out to a{" "}
+              <Link to="/community/community-managers">Community Manager</Link>
+            </p>
+
+            <h4 id = "program-participation">Q: What are Participation Required Programs?</h4>
+            <p>
+              A: Participation Required projects are by invite-only. Here’s how to
+              get your invite: demonstrate consistent engagement in the
+              community, passionate and technical ability to significantly
+              impact projects. Community Managers, MeshMates, and Maintainers
+              meet once a week to review outstanding individuals in the
+              community and extend invites to those contributors. An invitation
+              to participate in one of these programs entails a two week tryout
+              period in which you are given training and support, while you join
+              core team meetings to self-assess your desire to join the program
+              long-term and both self-assess and demonstrate your ability to be
+              a net positive contributor to the program. Program participants
+              are very well supported, learn and grow quickly. Their works are
+              often lauded on conference stages with bright spotlights placed on
+              their feats. These programs are intense, but amazing. Each a
+              participant is commits to maintain a specific level of daily
+              participation. If you are interested to explore, please inquire
+              with a <Link to="/community/community-managers">Community Manager</Link>, <Link to="/community/meshmates">MeshMate</Link>
```

Comment:
- No space in the preview between `meshmate` and `or`
```suggestion
              with a <Link to="/community/community-managers">Community Manager</Link>, <Link to="/community/meshmates">MeshMate </Link>
```

Reply:
@Mohith234 Understood, I will do the needful.